client/app/types/store.ts

```diff
@@ -20,6 +20,8 @@ import { AdminState } from 'bundles/system/admin/admin/types';
 import { InstanceAdminState } from 'bundles/system/admin/instance/instance/types';
 import { GlobalUserState } from 'bundles/users/types';
 
+import { TimelinesData } from './course/referenceTimelines';
```

Comment:
I think it's better to create a separate type (TimelinesState) for the timeline store's state (ie reference-timelines/store/types.ts) since the state of the store is not necessarily the same as the data type received from the backend (TimelinesData)